advance_ai_agents/candidate analyser/main.py

```
@@ -0,0 +1,202 @@
+import streamlit as st
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)